2021-05-17-announcements.md

```diff
@@ -0,0 +1,34 @@
+# CFDE Announcements
+
+## Cross-Pollination Events
+The next event will be on **Tuesday, June 1, at 11am PDT / 2pm EDT**. Presentations from the [CFDE-CC and SPARC](https://docs.google.com/document/d/1CFpYz4lPG7Bktf_4ajSMME3ERNNO_VNHYPJOqcElNc0/edit?usp=sharing) will detail the CurIndex and the Connectivity Knowledge Base. Regular Cross-Pollination events take place on the first Tuesday of every month. Please check out the [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) and let us know if you would like to present! We are actively looking for speakers for our July and August meetings. Please contact us today if you are interested in presenting at support@cfde.atlassian.net.
+
+## CFDE Trainings and Workshops
+Are you interested in Common Fund data? Come see a 1-hour pop-up demo of newly released CFDE Data Portal that allows you to search across data from HuBMAP, Kids First Data Resource Center, Metabolomics, LINCS, exRNA, GTEx, and HMP from a single website! [Register here](https://www.nih-cfde.org/events/cfde-training-workshop/?pk_campaign=anc). Please check out the [CFDE Events Calendar](https://www.nih-cfde.org/events/?pk_campaign=anc) to find and register for future events. Our workshops are open to all, so feel free to share this information with interested friends and colleagues.
```

Comment:
"Come see a 1-hour pop-up demo on **Wednesday, May 26, at 10am PDT/ 1pm EDT** of the ..."